invenio_accounts/config.py

```diff
@@ -31,3 +31,7 @@
 # ACCOUNTS_FORCE_BLUEPRINT_REGISTRATION can be use to override
 # ``register_blueprint`` option.
 ACCOUNTS_REGISTER_BLUEPRINT = None
+
+# Set the secure flag on the session cookie to ensure it is only transmitted
+# over a secure channel.
+SESSION_COOKIE_SECURE = True
```

Comment:
What about just creating a warning before first request if the `SESSION_COOKIE_SECURE` is `False` and we are not in debug or testing mode?